src/component/base/DraftEditorProps.js

```diff
@@ -48,12 +48,12 @@ export type DraftEditorProps = {
   blockRendererFn?: (block: ContentBlock) => ?Object,
 
   // Function that returns a cx map corresponding to block-level styles.
-  blockStyleFn?: (type: number) => string,
+  blockStyleFn?: (block: ContentBlock) => string,
 
   // A function that accepts a synthetic key event and returns
   // the matching DraftEditorCommand constant, or null if no command should
   // be invoked.
-  keyBindingFn: (e: SyntheticKeyboardEvent) => ?string,
+  keyBindingFn: (e: SyntheticKeyboardEvent) => ?DraftEditorCommand,
```

Comment:
I'm not sure this is entirely correct. `getDefaultKeyBinding` will return a `DraftEditorCommand`, but one might define a custom key binding function that returns a different arbitrary command string.